docs/organizer-guide/confs/community-events.rst

```diff
@@ -0,0 +1,20 @@
+Community Events
+================
+
+We are happy to promote events that are created by our community.
+They are **not** officially organized by Write the Docs organizers,
+but they are sister events that share a similar goal.
+
+Requirements
+------------
+
+In order to be listed as a community event,
+we require a few things.
+This is primarily to make sure people understand who is organizing the event,
+and to make sure companies can't run events inside our community for their own profit.
+
+Community events:
```

Comment:
Hmmmmmm I'm wondering about the name "community event", coz theoretically we run those as well. Let's talk about this at the call today, I'm thinking affiliate or something more accurate but less stuffy.